docs/contracts.rst

```diff
@@ -584,6 +587,87 @@ Please ensure you test your fallback function thoroughly to ensure the execution
         }
     }
 
+.. _overload-function:
+
+Function Overloading
+====================
+A Contract can have multiple functions of the same name but with different arguments.
```

Comment:
Please also mention that return parameters are not taken into considering for overloading. 

Simple reasoning for this: "Which version of an overloaded function to use if it is an statement without assigning its return values?"

Reply:
I mention that in the "Function Resolution" sub section further down.
https://github.com/ethereum/solidity/pull/3218/files#diff-754689a291c0a19b500c31eb6c1d30c7R649